src/components/ProjectCreateEditPage/ProjectCreateEditPage.tsx

```diff
@@ -276,9 +302,16 @@ export const ProjectCreateEditPage = ({ editPage = false }: IProjectCreateEditPa
       <Flex direction={flexDirection}>
         <FlexItem>
           {editPage ? (
-            <DataContainer {...dataContainerCreateEdit} title="Edit Project">
-              {formComponent}
-            </DataContainer>
+            isPatching ? (
+              <ServiceContainerCreating {...dataContainerEditPatch} title="Edit Project">
```

Comment:
let's rename this from `ServiceContainerCreating` to `ServiceContainerCreatingUpdating`